### PR TITLE
feat: migrate dvsa/dvsa-doctrine-module to dvsa/mot-doctrine-module

### PR DESCRIPTION
**Pull request by jennifer-choo**
## Description

Related issue: [JIRA_TICKET_NUMBER](LINK_TO_JIRA_TICKET)

## Before submitting (or marking as "ready for review")

- [x] Does the pull request title follow the [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/) specification?
- [x] Have you performed a self-review of the code
- [x] Have you have added tests that prove the fix or feature is effective and working
- [x] Did you make sure to update any documentation relating to this change?

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+## Description
```
Comment (jennifer-choo) [inline .github/pull_request_template.md:1]: New file

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    uses: dvsa/.github/.github/workflows/php-library-tests.yml@v3.2.3
+    with:
+      php-versions: "[\"8.0\", \"8.1\"]"
+
+  security:
+    uses: dvsa/.github/.github/workflows/php-library-security.yml@v3.2.3
+    with:
+      php-versions: "[\"8.0\", \"8.1\"]"
+    secrets:
+      SNYK_TOKEN: ${{ secrets.SNYK_TOKEN }}
```
Comment (jennifer-choo) [inline .github/workflows/ci.yml:29]: New file

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
+# How to contribute
```
Comment (jennifer-choo) [inline CONTRIBUTING.md:1]: New file

File: captainhook.json [hunk 4]
```diff
@@ -0,0 +1,27 @@
+{
```
Comment (jennifer-choo) [inline captainhook.json:1]: New file

File: phpcs.xml [hunk 5]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0"?>
```
Comment (jennifer-choo) [inline phpcs.xml:1]: New file

File: phpstan.neon [hunk 6]
```diff
@@ -0,0 +1,7 @@
+parameters:
```
Comment (jennifer-choo) [inline phpstan.neon:1]: New file

File: psalm.xml [hunk 7]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
```
Comment (jennifer-choo) [inline psalm.xml:1]: New file

File: vendor-bin/psalm/composer.json [hunk 8]
```diff
@@ -0,0 +1,9 @@
+{
```
Comment (jennifer-choo) [inline vendor-bin/psalm/composer.json:1]: New file

File: vendor-bin/phpstan/composer.json [hunk 9]
```diff
@@ -0,0 +1,9 @@
+{
+  "require": {
+    "php": "^8.0",
+    "phpstan/phpstan": "^1.10"
+  },
+  "config": {
+    "preferred-install": "dist"
+  }
+}
```
Comment (jennifer-choo) [inline vendor-bin/phpstan/composer.json:9]: New file

File: vendor-bin/psalm/composer.json [hunk 10]
```diff
@@ -0,0 +1,9 @@
...
+  "require": {
+    "php": "^8.0",
+    "vimeo/psalm": "^5.12"
+  },
+  "config": {
+    "preferred-install": "dist"
+  }
+}
```
Comment (jennifer-choo) [inline vendor-bin/psalm/composer.json:9]: New file

File: vendor-bin/php-codesniffer/composer.json [hunk 11]
```diff
@@ -0,0 +1,10 @@
+{
+  "require": {
+    "php": "^8.0",
+    "squizlabs/php_codesniffer": "^3.7",
+    "dvsa/coding-standards": "^2.0"
+  },
+  "config": {
+    "preferred-install": "dist"
+  }
+}
```
Comment (jennifer-choo) [inline vendor-bin/php-codesniffer/composer.json:10]: New file

File: composer.json [hunk 12]
```diff
@@ -0,0 +1,39 @@
...
+            "phpunit"
+        ]
+    },
+    "config": {
+        "allow-plugins": {
+            "laminas/laminas-dependency-plugin": true,
+            "captainhook/plugin-composer": true
+        }
+    }
+}
```
Comment (jennifer-choo) [inline composer.json:39]: Changed: renamed package name and updated doctrine/cache package vulnerability caught on snyk

File: LICENSE [hunk 13]
```diff
@@ -0,0 +1,21 @@
+MIT License
```
Comment (jennifer-choo) [inline LICENSE:1]: New file

File: .github/pull_request_template.md [hunk 14]
```diff
@@ -0,0 +1,14 @@
...
+Include a summary of the change here.
+-->
+
+Related issue: [JIRA_TICKET_NUMBER](LINK_TO_JIRA_TICKET)
+
+## Before submitting (or marking as "ready for review")
+
+- [ ] Does the pull request title follow the [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/) specification?
+- [ ] Have you performed a self-review of the code
+- [ ] Have you have added tests that prove the fix or feature is effective and working
```
Comment (caoimheb) [inline .github/pull_request_template.md]:
```suggestion
- [ ] Have you performed a self-review of the code?
- [ ] Have you have added tests that prove the fix or feature is effective and working?
```

**Review by caoimheb — APPROVED** [review]

**Review by faggc — APPROVED** [review]

**Review by DariuszJerzewski — APPROVED** [review]

**Review by DariuszJerzewski — APPROVED** [review]

**Pull request merged**